api/admin.go

```diff
@@ -112,7 +113,7 @@ func getAllAudits(c *Context, w http.ResponseWriter, r *http.Request) {
 		audits := result.Data.(model.Audits)
 		etag := audits.Etag()
 
-		if HandleEtag(etag, w, r) {
+		if HandleEtag(etag, "Get All Audits", w, r) {
```

Comment:
At first I didn't' realize these were Prometheus labels.  I was expecting to see somethings like `get_all_audits` since this data may be pumped via ETL into other system I figured the spaces would be a problem (like a db column name).  I assume you just copied a Prometheus example so it must be OK.